src/Nox.Generator/Application/ServiceCollectionExtensions.template.cs

```diff
@@ -41,6 +41,12 @@ public static IServiceCollection AddNox(this IServiceCollection services, WebApp
             configurator.WithMessagingTransactionalOutbox<AppDbContext>();
             configureNox?.Invoke(configurator);
         });
+        services.AddScoped(typeof(Nox.Domain.IRepository), serviceProvider =>
```

Comment:
this is enough, since all are registered in the container
```suggestion
        services.AddScoped<Nox.Domain.IRepository,Nox.Domain.Repository>
```

Reply:
it is thrown the following exception if I change it to the suggestion:
`Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'Nox.Domain.Repository'`